sklearn/manifold/_locally_linear.py

```diff
@@ -368,7 +368,7 @@ def locally_linear_embedding(
                                                  U[:, k:n_components])
                 j += n_components - k
 
-            Q, R = qr(Yi)
+            Q, R = qr(Yi, check_finite=False)
```

Comment:
As mentioned in your commit message we assume that `eigh` call does not return non finite values. I do not know if this is true but someone else might.